Domain/Sources/Domain/extension/UserMapper.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  File.swift
+//  
+//
+//  Created by Hassan on 25/03/2023.
+//
+
+import Foundation
+import Core
+public struct UserMapper {
+    public func userCodableToUser(_ inuser: UserCodable) -> User {
```

Comment:
rename to transformCodableToUser ,
 rename inuser to model 
and rename folder extension to Mapper

Reply:
done